jena-core/src/test/java/org/apache/jena/graph/GraphWithPerformContractTest.java

```diff
@@ -68,7 +68,13 @@ public void testPerformAdd_Triple() {
 		g.performAdd(triple("S3 P3 O3"));
 		txnCommit(g);
 		GL.assertEmpty();
+		txnRun( g, new Runnable() {
```

Comment:
There is no need to explicitly have a `Runnable` here or in the other uses:
```
txnRun( g, ()->assertTrue(g.contains(triple("S3 P3 O3"))) );
```
makes the tests shorter and clearer.